src/main/java/org/justinbaur/bankteller/runner/TellerTerminalRunner.java

```diff
@@ -104,11 +116,58 @@ private void withdraw() {
     }
 
     private void listMenu() {
-        System.out.println("\nPlease follow the list of commands:");
+        System.out.println(String.format("\nCurrently logged in as: %s", cAcct.getName()));
+        System.out.println("Please follow the list of commands:");
         System.out.println("  balance - to check your current balance.");
         System.out.println("  deposit - to place money into your account.");
         System.out.println("  withdraw - to take money out of your account.");
-        System.out.println("  exit - to shut down your session.");
+        System.out.println("  logout - to log out of the current account.");
+        System.out.println("  exit - completely exit the application.");
         System.out.print("->");
     }
+
+    private HashMap<Integer, AccountServiceInMemoryImpl> populateAccounts(){
+        Integer[] accountIDList = new Integer[] {0001, 1000, 1001, 1002, 1003, 1004};
+        String[] accountUsernameList = new String[] {"ADMIN", "Dustin D", "Justin B", "Boydvan L", "Gino T", "Glenn D"};
+        Integer[] accountStartingBalanceList = new Integer[] {999999, 15, 55000, 55001, 236, 1450};
+
+        HashMap<Integer, AccountServiceInMemoryImpl> accountLookup = new HashMap<Integer, AccountServiceInMemoryImpl>();
+
+        try {
+            for (int i = 0; i < accountIDList.length; i++ ) {
+                AccountServiceInMemoryImpl newAccount = new AccountServiceInMemoryImpl(accountIDList[i], accountUsernameList[i]);
+                newAccount.addBalance(accountStartingBalanceList[i]);
+                accountLookup.put(accountIDList[i], newAccount);
+            }
+        } catch (ArrayIndexOutOfBoundsException e) {
+            e.printStackTrace();
```

Comment:
It's poor practice to dump the entire stack trace, try putting some context around the error message that occurred.